src/main/java/com/esri/samples/symbology/read_symbols_from_mobile_style_file/ReadSymbolsFromMobileStyleFileController.java

```diff
@@ -0,0 +1,353 @@
+                    break;
+                  case "Eyes":
+                    eyesSelectionListView.getItems().add(symbolStyleSearchResult);
+                    break;
+                  case "Mouth":
+                    mouthSelectionListView.getItems().add(symbolStyleSearchResult);
+                    break;
+                  default:
+                    break;
+                }
+              });
+
+              // create the symbol to populate the preview
+              buildCompositeSymbol();
+
+            } catch (InterruptedException | ExecutionException e) {
+              new Alert(Alert.AlertType.ERROR, "Error performing the symbol search" + e.getMessage()).show();
+            }
+          });
+        } catch (InterruptedException | ExecutionException e) {
+          new Alert(Alert.AlertType.ERROR, "Error retrieving default search parameters for symbol search" + e.getMessage()).show();
+        }
+      });
+    });
+  }
+
+  /**
+   * Gets the keys of the selected symbol components, and constructs a multilayer point symbol.
+   */
+  @FXML
+  private void buildCompositeSymbol() {
+
+    // retrieve the requested keys for the selected hat, eyes and mouth symbols
+    String hatKey = getRequestedSymbolKey(hatSelectionListView);
+    String eyesKey = getRequestedSymbolKey(eyesSelectionListView);
+    String mouthKey = getRequestedSymbolKey(mouthSelectionListView);
+
+    // create a list of keys to use for getting the requested symbol.
+    List<String> symbolKeys = Arrays.asList("Face1", eyesKey, mouthKey, hatKey);
+
+    // get the symbol from the SymbolStyle
+    ListenableFuture<Symbol> symbolFuture = emojiStyle.getSymbolAsync(symbolKeys);
+    symbolFuture.addDoneListener(() -> {
+      try {
+        faceSymbol = (MultilayerPointSymbol) symbolFuture.get();
+        if (faceSymbol == null) {
+          return;
+        }
+
+        // loop through all the symbol layers and lock the color
+        faceSymbol.getSymbolLayers().forEach(symbolLayer -> symbolLayer.setColorLocked(true));
+
+        // unlock the color of the base layer. Changing the symbol layer will now only change this layer's color
+        faceSymbol.getSymbolLayers().get(0).setColorLocked(false);
+
+        // set the color of the symbol
+        faceSymbol.setColor(colorSelectionListView.getSelectionModel().getSelectedItem());
+
+        // set the size of the symbol
+        faceSymbol.setSize((float) sizeSlider.getValue());
+
+        // update the symbol preview
+        updateSymbolPreview(faceSymbol);
+
+      } catch (ExecutionException | InterruptedException e) {
+        new Alert(Alert.AlertType.ERROR, "Error creating symbol with the provided symbol keys" + e.getMessage()).show();
+      }
+    });
+  }
+
+  /**
+   * Returns the symbol key of the symbol style search result that is currently selected in the list view.
+   * @param listView the list view of which to query the selected item
```

Comment:
I think it's normal (at least it is in our API code) to have a blank line before the tags, like you have in `updateSymbolPreview` below